src/main/java/com/game/Battleship/models/Board.java

```diff
@@ -0,0 +1,44 @@
+package com.game.Battleship.models;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+
+    private int size;
+    private List<Cell> boardField;
+
+    public Board(){
+        this.size = 10;
+        this.boardField = generateBoardField();
+    }
+
+    private List<Cell> generateBoardField(){
+        List<Cell> result = new ArrayList<>();
+        for(int i=1; i<=this.size; i++){
```

Comment:
What are these two for loops representing? Is there a better way to communicate that here?

Reply:
Done